napari_covid_if_annotations/gui.py

```diff
@@ -1,64 +1,34 @@
 import os
-from magicgui import magicgui, register_type
-from napari import Viewer
-from qtpy import QtWidgets
-from ._key_bindings import (update_layers,
-                            toggle_hide_annotated_segments,
-                            paint_new_label,
-                            _save_labels)
-
-#
-# additional gui elements
-#
-
-
-def get_viewers(gui, *args):
-    try:
-        viewer = gui.parent().qt_viewer.viewer
-        return (viewer,)
-    except AttributeError:
-        return tuple(v for v in globals().values() if isinstance(v, Viewer))
-
-
-register_type(Viewer, choices=get_viewers)
-
-
-# TODO expose the partial parameter to the gui
-@magicgui(call_button='save annotations [shfit + s]', viewer={'visible': False})
-def save_gui(viewer: Viewer):
-    _save_labels(viewer)
-
-
-@magicgui(call_button='update layers [u]', viewer={"visible": False})
-def update_layers_gui(viewer: Viewer):
-    update_layers(viewer)
-
-
-@magicgui(call_button='hide annotated cells [h]', viewer={"visible": False})
-def toggle_hide_annotated_segments_gui(viewer: Viewer):
-    toggle_hide_annotated_segments(viewer)
-
-
-@magicgui(call_button='get next label [n]', viewer={"visible": False})
-def paint_new_label_gui(viewer: Viewer):
-    paint_new_label(viewer)
+from qtpy.QtWidgets import QLabel, QPushButton
+from ._key_bindings import (
+    update_layers,
+    toggle_hide_annotated_segments,
+    paint_new_label,
+    _save_labels,
+)
 
 
 def connect_to_viewer(viewer):
     """ Add all gui elements to the viewer
     """
 
-    # get gui buttons for the additional functionality
-    saving_gui = save_gui.Gui()
-    update_gui = update_layers_gui.Gui()
-    hide_gui = toggle_hide_annotated_segments_gui.Gui()
-    paint_gui = paint_new_label_gui.Gui()
+    save_gui_btn = QPushButton("save annotations [shfit + s]")
+    save_gui_btn.clicked.connect(lambda: _save_labels(viewer))
+
+    update_gui_btn = QPushButton("update layers [u]")
+    update_gui_btn.clicked.connect(lambda: update_layers(viewer))
+
+    hide_gui_btn = QPushButton("hide annotated cells [h]")
+    hide_gui_btn.clicked.connect(lambda: toggle_hide_annotated_segments(viewer))
+
+    paint_gui_btn = QPushButton("get next label [n]")
+    paint_gui_btn.clicked.connect(lambda: paint_new_label(viewer))
 
     # make a tooltop about the label colors
-    tooltip = QtWidgets.QLabel()
+    tooltip = QLabel()
 
     # TODO see if we can get this working
```

Comment:
if you want help showing this image, let me know!

Reply:
I think I can get this to work myself, I just didn't have time to look into it yet and make the actual image for it.
Let's leave it out of this PR for now, I will look into it once I have time and let you know if I run into any isses.